core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -73,21 +74,30 @@ public void initializeLogging(LogConfig config, final List<RuntimeValue<Optional
         final Logger rootLogger = logContext.getLogger("");
 
         rootLogger.setLevel(config.level.orElse(Level.INFO));
+
+        ErrorManager errorManager = new OnlyOnceErrorManager();
+        final Map<String, CleanupFilterConfig> filters = config.filters;
+        List<LogCleanupFilterElement> filterElements = new ArrayList<>(filters.size());
+        for (Entry<String, CleanupFilterConfig> entry : filters.entrySet()) {
+            filterElements.add(new LogCleanupFilterElement(entry.getKey(), entry.getValue().ifStartsWith));
+        }
+
+        Map<String, Handler> namedHandlers = createNamedHandlers(config, possibleFormatters, errorManager, filterElements);
```

Comment:
The console handler has to be created before the named handlers, otherwise the error manager is not correctly set up in time.  AFAICT this didn't need to be factored out to a separate method.

Reply:
Moved the creation of the named handlers after creation of the handlers at root level.